src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.Android.cs

```diff
@@ -32,8 +32,17 @@ public TextBoxView()
 
 			_isInitialized = true;
 
-            // This ensures the TextBoxView gets resized as Text changes
-            LayoutParameters = new Android.Views.ViewGroup.LayoutParams(
+			this.KeyDown += (s, e) =>
+			{
+				Console.WriteLine("!");
+			};
+			this.KeyUp += (s, e) =>
+			{
+				Console.WriteLine("!");
```

Comment:
Exciting though it may be, don't merge this :p

Reply:
Good catch.

Reply:
Fixed